Assets/Scripts/BulletManagement/LimitedEmitter.cs

```diff
@@ -0,0 +1,25 @@
+﻿using UnityEngine;
+
+namespace BulletManagement
+{
+    // An emitter that emits a fixed number of bullet cycles and then destroys itself
+    public class LimitedEmitter : Emitter
+    {
+        // The number of bullet waves to emit
+        public int emitCycles = 1;
+
+        // Running total of emitted bullet waves
+        private int _cyclesEmitted = 0;
```

Comment:
Should private variables have underscores? Do we want to navigate to this kind of style? I've been doing none-under score. Perhaps we can just stick with normal variable conventions for now?

Reply:
I think this is only one of two variables where I've done this. I've always been taught that this is the way private fields are supposed to be named in C#, but I don't really care. I can rename them.